examples/semantic_segmentation/semantic_segmentation/data_loader.py

```diff
@@ -40,40 +38,33 @@ class ResultMask:
 
 
 ResultMasks = Tuple[ResultMask, ResultMask, ResultMask]
-ResultMasksByLocator = Tuple[str, ResultMask, ResultMask, ResultMask]
 
 
 class DataLoader:
     def __init__(self):
-        self.pool = ThreadPoolExecutor()
+        self.pool = ThreadPoolExecutor(max_workers=32)
```

Comment:
Is there a reason we specify max_workers here and not in ActivationMapUploader?

Reply:
Ya, DataLoader future is IO bound since it is downloading two assets and binarizing the mask(relatively quick), while Activation map uploader is CPU bound due to `create_bitmap`.
ActivationMapUploader is not benefitting much from using thread pools and should instead utilize multiprocessing since it is CPU bound work that can be parallelized.